README.md

```diff
@@ -508,6 +508,23 @@ extra conditions contradict the conditions in the rules, the Transfer can never
 The Receiver must post a comment describing the resolution of these extra conditions with validity as a result, before
 he can accept the Transfer.
 
+**384** *Subgamification*
+
+Any subset of 2 or more players may agree to play any game in real life with points as a stake. 
+The subset of players must verbally agree on the stakes and a division of points based on the result of game before starting it. 
+The total decrease in points must be be one less than the total increase in points. 
+The division must be such that no player can win Nomic or lose more points than they have, regardless
+of the games result.
+
+Any of the players that gained the most points will submit a PR reflecting the agreed changes in points.
+This PR may be merged by anyone after being open for 24 hours, if and only if above conditions are still
+true at the time of merging. This PR does not need to be voted on. 
```

Comment:
Can people object to it though?

Reply:
Sure. That's why there's a time limit. I didn't feel the need to include a specific appeal process, since any objection would be based on "you're not following the rules" for which there would be a trial if someone merged it anyway.